scripts/install-fmt.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+# number of jobs to run
+NPROCS=1
+# Find available processor numbers based on different OS.
+OS="`uname -s`"
+case $OS in
+  'Linux')
+    NPROCS="$(grep -c ^processor /proc/cpuinfo)"
+    ;;
+  'Darwin')
+    NPROCS="$(sysctl -n hw.ncpu)"
+    ;;
+esac
+
+wget https://github.com/fmtlib/fmt/archive/refs/tags/10.1.1.tar.gz -O - | tar zxf - &&
+  cd fmt-10.1.1/ &&
+  mkdir build/ &&
+  cmake -B build/ . &&
+  sudo make -j${NPROCS} -C build/ install &&
+  cd .. &&
+  rm -r fmt-10.1.1/
```

Comment:
I'm surprised that `rm` is requesting permission to remove some write-protected files:
```
rm: remove write-protected regular file 'fmt-10.1.1/build/install_manifest.txt'? y
rm: remove write-protected regular file 'fmt-10.1.1/build/libfmt.a'? y
rm: remove write-protected regular file 'fmt-10.1.1/build/test/libtest-main.a'? y
rm: remove write-protected regular file 'fmt-10.1.1/build/test/gtest/libgtest.a'? y
rm: remove write-protected regular file 'fmt-10.1.1/build/test/gtest/CMakeFiles/gtest.dir/gmock-gtest-all.cc.o'? y
rm: remove write-protected regular file 'fmt-10.1.1/build/test/gtest/CMakeFiles/gtest.dir/gmock-gtest-all.cc.o.d'? y
...
```
This behavior might be due to these files being generated by `sudo make -j${NPROCS} -C build/ install`, which uses root privileges. I'm wondering whether a similar situation occurs when installing _qbe_ and _cxxopts_.